### PR TITLE
Created basic project structure.

### PR DESCRIPTION
**Pull request by nick79**
Created basic project structure.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import os
+
```
Comment (comanjoni) [inline app.py]: Remove this empty space. It's not by PEP8.

Comment (comanjoni) [inline app.py]: Remove empty space between imports. It's not by PEP8.

**Review by comanjoni — APPROVED** [review]
Just make sure that you resolve my comment.

**Comment by nick79** [comment]
Merged.

**Pull request merged**